index.html

```diff
@@ -1,262 +1,359 @@
+                ><img src="./Graphics/medium.svg" alt="medium"
+              /></a>
+            </li>
+            <li class="icon github">
+              <a href="https://github.com/calebchris000"
+                ><img src="./Graphics/github.svg" alt="github"
+              /></a>
+            </li>
+
+            <li class="icon peace">
+              <a href="https://www.jw.org/en">
+                <img src="./Graphics/Vector.svg" alt="peace"
+              /></a>
+            </li>
+          </ul>
+
+          <p class="work get-resume">Get my resume</p>
+        </ul>
+      </section>
+      <div class="space"></div>
+
+      <section>
+        <ul class="language-header">
+          <li>Languages</li>
+          <li>
+            <img
+              class="toggle"
+              src="./Graphics/Normal Button/Tertiary/Icons/toggle.svg"
+              alt=""
+            />
+          </li>
+        </ul>
+
+        <ul class="languages">
+          <div class="js-container">
+            <img
+              src="./Graphics/Normal Button/Tertiary/Icons/javascript.png"
+              alt=""
+            />
+            <li class="javascript">JavaScript</li>
+          </div>
+
+          <div class="html-container">
+            <img
+              src="./Graphics/Normal Button/Tertiary/Icons/html.png"
+              alt=""
+            />
+            <li class="html5">HTML</li>
+          </div>
+
+          <div class="css-container">
+            <img src="./Graphics/Normal Button/Tertiary/Icons/css.png" alt="" />
+            <li class="css3">CSS</li>
+          </div>
+        </ul>
+
+        <ul class="frameworks-header">
+          <li class="frameworks">Frameworks</li>
+          <li>
+            <img
+              class="toggle-right"
+              src="./Graphics/Normal Button/Tertiary/Icons/toggle-right.svg"
+              alt=""
+            />
+          </li>
+        </ul>
+        <div class="line"></div>
+
+        <ul class="skills-header">
+          <li class="frameworks">Skills</li>
+          <li>
+            <img
+              class="toggle-right"
+              src="./Graphics/Normal Button/Tertiary/Icons/toggle-right.svg"
+              alt=""
+            />
+          </li>
+        </ul>
+        <div class="line"></div>
+      </section>
+
+      <section>
+        <div class="contact-container">
+          <h1>Contact me</h1>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <form action="https://formspree.io/f/mlekdbdj" method="post">
+            <ul>
+              <li>
+                <label for="name"></label>
+                <input
+                  type="text"
+                  id="name"
+                  name="user_name"
+                  placeholder="Yeremias"
+                  required
```

Comment:
- [ ] Kindly you may consider adding some HTML validation to your name field where it says It is supposed to have a `maxlength = 30`